### DISCUSSION TITLE
[Solved] Crazyflie client not working on Raspberry Pi and Mac OS Sonoma

### DISCUSSION THREAD

**Discussion by ataffanel** [bitcraze/discussions#1218] [Announcements]
We have noticed recently that the state of the client has quickly degraded on two platforms, the Raspberry Pi and Mac OS.

 Both problem have been added to the issue tracker: bitcraze/crazyflie-clients-python#698 and bitcraze/crazyflie-clients-python#694

So we are aware of the problem. When we have time to work on it we will update the issues and this discussion thread. If anyone wants to help please heads up to the issues and we can discuss there. The raspberry pi has a clear path forward, we are open to switching to PySide, but we currently have no idea how to approach the Mac issue (none of us at Bitcraze uses a Mac in a daily basis). So both issue are prefect to contribute!

**Comment by knmcguire** [comment]
For Apple M1 there seems to be a temp fix for it: https://github.com/bitcraze/crazyflie-clients-python/issues/694

**Comment by ataffanel** [comment]
The latest release of the client and its [documentation](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/installation/install/#raspberrypy-bookworm) fixes the situation, the client is now working find on mac and raspberrypi!
